packages/trader/src/App/Components/Elements/PositionsDrawer/result-mobile.jsx

```diff
@@ -5,16 +5,10 @@ import { CSSTransition } from 'react-transition-group';
 import { Icon } from '@deriv/components';
 import { localize } from '@deriv/translations';
 
-class ResultMobile extends React.PureComponent {
-    handleClick = e => {
-        if (this.props.is_unsupported) {
-            e.preventDefault();
-            this.props.onClick();
-        }
-    };
-
-    render() {
-        const { is_visible, result } = this.props;
+const ResultMobile = ({
+    is_visible,
+    result,
+}) => {
```

Comment:
Since certain props are being removed, you may need to look into places in which we pass them to the components and change them accordingly.

Reply:
sure, will check